chainer/link_hook.py

```diff
@@ -62,7 +62,7 @@ class LinkHook(object):
     :meth:`~chainer.LinkHook.forward_preprocess`
     (resp. :meth:`~chainer.LinkHook.forward_postprocess`)
     of all link hooks registered to this link are called before (resp. after)
-    :meth:`Link.forward() <chainer.Link.forward>` method of the link.
+    :meth:`~chainer.Link.forward` method of the link.
```

Comment:
Please revert to the previous markup. This would truncate the whole prefix to leave just `forward`.